packages/twenty-front/src/modules/databases/graphql/queries/createOneDatabaseConnection.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from '@apollo/client';
```

Comment:
Could you create a directory "mutations", sibling to "queries", and move this file there?